src/assisted_test_infra/test_infra/utils/logs_utils.py

```diff
@@ -140,39 +161,36 @@ def _verify_oc_logs_uploaded(cluster_tar_path):
 
 
 def _verify_node_logs_uploaded(dir_path, file_path):
-    gz = tarfile.open(os.path.join(dir_path, file_path))
-    logs = gz.getnames()
-    for logs_type in ["agent.logs", "installer.logs", "mount.logs"]:
-        assert any(logs_type in s for s in logs), f"{logs_type} isn't found in {logs}"
-    gz.close()
+    with tarfile.open(os.path.join(dir_path, file_path)) as gz:
```

Comment:
Why not use `_safe_open_tar`?

Reply:
because it doesn't do extraction, only reads the names and asserts their values